#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_freetype_makefile.dpatch by  <gaudenz@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix Includes for building libps.a

@DPATCH@
diff -urNad passepartout-0.6-3~/src/ps/Makefile.am passepartout-0.6-3/src/ps/Makefile.am
--- passepartout-0.6-3~/src/ps/Makefile.am	2004-10-07 14:55:52.000000000 +0200
+++ src/ps/Makefile.am	2005-12-21 03:59:09.000000000 +0100
@@ -10,7 +10,7 @@
 examinepdf_SOURCES = examinepdf.cc
 
 # for Glib::ustring
-libps_a_CXXFLAGS = $(GTKMM_CFLAGS)
+libps_a_CXXFLAGS = $(GTKMM_CFLAGS) $(FREETYPE_CFLAGS)
 
 libps_a_CXXFLAGS += -I$(top_srcdir)/src
 
